src/qae/object-properties.js

```diff
@@ -16,20 +16,14 @@ const properties = {
   qHyperCubeDef: {
     /**
      * The maximum amount of dimensions is 1000
-     * @type {DimensionProperties[]}
+     * @type {DimensionProperty[]}
```

Comment:
For one dimension, it still is a collection of properties. I think this should be reverted

Reply:
Sure

Reply:
Done